torch/_functorch/aot_autograd.py

```diff
@@ -436,10 +436,15 @@ class ViewAndMutationMeta:
     # another user output (in both cases, we won't redundantly append bases to the end of the graph)
     num_intermediate_bases: int
 
+    # For inference only: instructs us to keep data-only input mutations directly in the graph
+    keep_input_mutations: int
```

Comment:
It would be better to have a default value here instead of making a backwards incompatible change.

```python
    keep_input_mutations: int = 0
```